doc/source/tutorials/emulator.rst

```diff
@@ -0,0 +1,153 @@
+-----------------------------------------
+
+This tutorial guides you through executing a pulse sequence with a one-qubit emulator using the QuTiP backend. We will first need to construct the pulse sequence we would like to run. This step is exactly the same as any other Qibolab platform. To keep this tutorial concise, we will construct a pulse sequence using the native gate  pulses from the platform that are specified in the runcard. A pulse sequence is a series of pulses that will be sent to the qubit.
+
+We first import the `PulseSequence` object from `qibolab.pulses` and initialize an instance.
+
+.. code-block:: python
+
+    from qibolab.pulses import PulseSequence
+
+    sequence = PulseSequence()
+
+Now, we retreive the native RX pulse and readout pulse of qubit 0 and add them to the pulse sequence.
+
+.. code-block:: python
+
+    pulse_x0 = emulator_platform.create_RX_pulse(qubit=0, start=0)
+    pulse_r0 = emulator_platform.create_qubit_readout_pulse(qubit=0, start=(170 / 4.5))
+
+    sequence.add(pulse_x0)
+    sequence.add(pulse_r0)
+
+We can plot the sequence to inspect it.
+
+.. code-block:: python
+
+    sequence.plot()
+
+.. image:: emulator/pulse_sequence.png
+    :width: 800
+    :align: center
+
+Next we import the module `ExecutionParameters` from `qibolab.execution_parameters` which is required as an input to the `execute_pulse_sequence` method of any Qibolab platform. The value of `ExecutionParameters.acquisition_type` set to `AcquisitionType.DISCRIMINATION` from `qibolab.execution_parameters` by default is the only option currently supported by the emulator. Let us execute the pulse sequence with 1000 shots.
+
+.. code-block:: python
+
+    from qibolab.execution_parameters import ExecutionParameters
+
+    options = ExecutionParameters(nshots=1000)
+    results = emulator_platform.execute_pulse_sequence(sequence, options=options)
+
+Note that for Qibolab device platforms, analogues of the following instrument methods have to be run prior to executing pulse sequence. These methods for the pulse simulator instrument are only kept for consistency and do not have any functionality.
+
+
+.. code-block:: python
+
+    emulator_platform.instruments["pulse_simulator"].connect()
+    emulator_platform.instruments["pulse_simulator"].setup()
+    emulator_platform.instruments["pulse_simulator"].start()
+
+Likewise, for Qibolab device platforms, analogues of the following instrument methods have to be run after using the device. These methods for the pulse simulator instrument are only kept for consistency and do not have any functionality.
+
+.. code-block:: python
+
+    emulator_platform.instruments["pulse_simulator"].stop()
+    emulator_platform.instruments["pulse_simulator"].disconnect()
+
+Once the simulation is complete, it returns the same result object as any other Qibolab device platform. We can therefore retrieve these results in the same way:
+
+.. code-block:: python
+
+    results[list(results.keys())[1]].samples
```

Comment:
A dedicated tutorial for the emulator should not be required, as it should follow the same interface as any other controller.

Thus, creating and playing a pulse sequence on the emulator should be the same as for Zurich, otherwise it will be incompatible with Qibocal (e.g., but not only).

Reply:
So I will simply remove all details about loading an emulator as well as creating/playing pulses. I assume that the rest of `doc/source/tutorials/emulator.rst` that you have not flagged out should be kept there?

Reply:
Fixed in 6f442c1ffbb9b110c9d3118cdeb67d354c19b42a